patches/chromium/feat_expose_url_chain_from_urlloader.patch

```diff
@@ -0,0 +1,38 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: deepak1556 <hop2deep@gmail.com>
+Date: Tue, 19 Nov 2024 17:52:08 +0900
+Subject: feat: expose url chain from urlloader
+
+The patch adds the url chain to the URLResponseHead struct. This is
+used to support the Response.url property in the Fetch API.
+
+Should be upstreamed.
```

Comment:
Not necessarily a blocker but I'd like to see some feedback from upstream as that potentially could result in subsequent changes to this PR if they want a slightly different API shape or approach or something!